cmd/mr_list_test.go

```diff
@@ -88,3 +88,33 @@ func Test_mrListFivePerPage(t *testing.T) {
 	t.Log(mrs)
 	require.Contains(t, mrs, "#1 Test MR for lab list")
 }
+
+func Test_mrFilterByTargetBranch(t *testing.T) {
+	t.Parallel()
+	repo := copyTestRepo(t)
+	cmd := exec.Command("../lab_bin", "mr", "list", "-t", "non-existing")
+	cmd.Dir = repo
+
+	b, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	mrs := strings.Split(string(b), "\n")
+	require.Equal(t, "PASS", mrs[0])
```

Comment:
"PASS" will always be present in the output (assuming non-zero exec which will show as an err to the exec call). Its better to assert that the length only has the test prints in it. I think there are 2? lines. so something like `require.Equal(t, 2, len(mrs))